kani-compiler/kani_queries/src/lib.rs

```diff
@@ -66,8 +66,7 @@ pub struct QueryDb {
     emit_vtable_restrictions: bool,
     json_pretty_print: bool,
     ignore_global_asm: bool,
-    /// When set, instructs the compiler to produce the symbol table for CBMC in goto binary format instead of JSON format.
```

Comment:
Can you keep the comment but update it? We should mention that the option runs `symtab2gb` as well.

Reply:
fixed